src/sqlancer/MainOptions.java

```diff
@@ -49,6 +49,12 @@
     @Parameter(names = "--password", description = "The password used to log into the DBMS")
     private String password = "sqlancer"; // NOPMD
 
+    @Parameter(names = "--host", description = "The host used to log into the DBMS")
+    private String host = "sqlancer"; // NOPMD
```

Comment:
Since these options are not supported by all DBMSs (yet): Would it be possible to insert a (static) check method that is called by every provider that does not support these? This is optional for this PR, but it might be a useful future improvement.

Reply:
For the `host`, I think that it may use 'localhost' is more appropriate. But for `port` I think it is more reasonable to use placeholders.  Or you can use placeholders for both parameters. After all, this is not the core part of the software. It just wants users to connect to the database with custom `host` and `port` parameters.

Reply:
> Since these options are not supported by all DBMSs (yet): Would it be possible to insert a (static) check method that is called by every provider that does not support these? This is optional for this PR, but it might be a useful future improvement.

We modify the description information to remind users that this value will be reset by the system. Is it feasible?
```
@Parameter(names = "--host", description = "The host used to log into the DBMS. If the user does not specify this value, it will be reset to localhost by the system")
    private String host = "sqlancer"; // NOPMD

@Parameter(names = "--host", description = "The host used to log into the DBMS. If the user does not specify this value, it will be reset to database system default port by the system")
    private String host = "sqlancer"; // NOPMD
```

In fact, another important function of placeholders is to facilitate us to determine whether the user has specified this parameter, so invalid values ​​should be used as placeholders. The value `sqlancer` is a good choice.  These two values ​​are actually essentially different from username and password. The default of host is `localhost`. The default of `port` is database parameter.

Reply:
I do not yet understand yet why we need any placeholders in the options. None of the other options have deliberately invalid default values, I believe. If you want to indicate that no port or host have been set, should they be `null`?

Reply:
> I do not yet understand yet why we need any placeholders in the options. None of the other options have deliberately invalid default values, I believe. If you want to indicate that no port or host have been set, should they be `null`?

Yes, null is also possible. But according to the previous design, does the system still have to reset the host and port to the corresponding to `localhost` and the default port of database respectively? If you think `null` is more appropriate, I am willing to modify it. LMK, please.

Reply:
Yes, I believe that either `null`, while also printing the default values when the help text is printed, or implementing them as DBMS-specific options (rather than in `MainOptions`) would be ways to achieve this.